for_motion_planner: true
robot_r: 0.2

display_loc_edt: true
display_loc_ogm: true
display_glb_edt: false
display_glb_ogm: false
vis_interval: 1
vis_height: 0.8

# profile the rms every time the visualization msg is published
profile_loc_rms: false
profile_glb_rms: false
log_name: "test.csv"

occupancy_threshold: 180

voxel_width: 0.1

local_size_x: 10
local_size_y: 10
local_size_z: 3

ogm:
  min_height: -10
  max_height: 10

wave:
  fast_mode: true
  cutoff_dist: 6

# params of hash table
hash:
  bucket_max: 10000

  block_max: 11997
